src/renderer/extensions/vueNodes/components/InputOutputSlot.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { mount } from '@vue/test-utils'
+import { createPinia } from 'pinia'
+import { describe, expect, it, vi } from 'vitest'
+import { createI18n } from 'vue-i18n'
+
+import enMessages from '@/locales/en/main.json'
+import { useDomSlotRegistration } from '@/renderer/core/layout/slots/useDomSlotRegistration'
+
+import InputSlot from './InputSlot.vue'
+import OutputSlot from './OutputSlot.vue'
+
+// Mock composable used by InputSlot/OutputSlot so we can assert call params
+vi.mock('@/renderer/core/layout/slots/useDomSlotRegistration', () => ({
+  useDomSlotRegistration: vi.fn(() => ({ remeasure: vi.fn() }))
+}))
+
+const mountWithProviders = (component: any, props: Record<string, unknown>) =>
```

Comment:
This at least gets rid of `any`, but I'd probably rather have the full type safety for component and props that the test-utils types have. 

```suggestion
const mountWithProviders = (component: InputSlot|OutputSlot, props: Record<string, unknown>) =>
```

Reply:
Or have two different factory methods, one for each type. WET/AHA style.